shenyu-admin/src/main/resources/mappers/shenyu-dict-sqlmap.xml

```diff
@@ -32,65 +32,89 @@
     </resultMap>
 
     <sql id="Base_Column_List">
-        id, `type`, dict_code, dict_name, dict_value, `desc`, sort, enabled, date_created, date_updated
+        id,
+        `type`,
+        dict_code,
+        dict_name,
+        dict_value,
+        `desc`,
+        sort,
+        enabled,
+        date_created,
+        date_updated
     </sql>
 
     <select id="selectById" parameterType="java.lang.String" resultMap="BaseResultMap">
-        select
-        <include refid="Base_Column_List"/>
-        from shenyu_dict
-        where id = #{id,jdbcType=VARCHAR}
+        SELECT
+              <include refid="Base_Column_List"/>
+         FROM shenyu_dict
+        WHERE id = #{id, jdbcType=VARCHAR}
     </select>
 
     <select id="selectByDictCodeAndDictName" resultMap="BaseResultMap">
-        select
-        <include refid="Base_Column_List"/>
-        from shenyu_dict
-        where dict_code = #{dictCode,jdbcType=VARCHAR} and dict_name = #{dictName,jdbcType=VARCHAR}
+        SELECT
+              <include refid="Base_Column_List"/>
+         FROM shenyu_dict
+        WHERE dict_code = #{dictCode, jdbcType=VARCHAR}
+          AND dict_name = #{dictName, jdbcType=VARCHAR}
     </select>
 
     <select id="findByType" parameterType="java.lang.String" resultMap="BaseResultMap">
-        select
-        <include refid="Base_Column_List"/>
-        from shenyu_dict where `type` = #{type,jdbcType=VARCHAR}
+        SELECT
+              <include refid="Base_Column_List"/>
+         FROM shenyu_dict
+        WHERE `type` = #{type, jdbcType=VARCHAR}
     </select>
 
     <select id="findByTypeBatch" resultMap="BaseResultMap">
-        select
-        <include refid="Base_Column_List"/>
-        from shenyu_dict where `type` in
-        <foreach collection="typeSet" item="type"
-                 open="(" separator="," close=")">
-            #{type,jdbcType=VARCHAR}
+        SELECT
+              <include refid="Base_Column_List"/>
+         FROM shenyu_dict
+        WHERE `type` IN
+        <foreach collection="typeSet" item="type" open="(" separator="," close=")">
+              #{type,jdbcType=VARCHAR}
         </foreach>
     </select>
 
     <delete id="delete" parameterType="java.lang.String">
-        delete from shenyu_dict
-        where id = #{id,jdbcType=VARCHAR}
+        DELETE FROM shenyu_dict
+        WHERE id = #{id,jdbcType=VARCHAR}
```

Comment:
The keywords `WHERE` are not right-aligned, It should be right-aligned with `FROM`.